api/krusty/configmaps_test.go

```diff
@@ -261,24 +261,13 @@ krypton
 xenon
 radon
 `)
-	m := th.Run(".", th.MakeDefaultOptions())
-	th.AssertActualEqualsExpected(m, `
-apiVersion: v1
-data:
-  fruit: apple
-  nobles: |2
-
-    helium
-    neon
-    argon
-    krypton
-    xenon
-    radon
-  vegetable: broccoli
-kind: ConfigMap
-metadata:
-  name: blah-bob-db529cg5bk
-`)
+	err := th.RunWithErr(".", th.MakeDefaultOptions())
+	if err == nil {
+		t.Fatalf("expected an error")
+	}
+	assert.Contains(t, err.Error(),
+		"kustomization unmarshal error: error converting YAML to JSON: yaml: unmarshal errors:\n"+
+			"  line 13: key \"literals\" already set in map\n  line 18: key \"files\" already set in map")
```

Comment:
be an error like [this TODO](https://github.com/kubernetes-sigs/kustomize/blob/e3981daf0ebca762fe1a29dfd39b70418c6697ba/api/krusty/configmaps_test.go#L231) comment.